src/NuGet.Core/Test.Utility/ProjectManagement/TestNuGetProjectContext.cs

```diff
@@ -25,7 +25,7 @@ public FileConflictAction ResolveFileConflict(string message)
             return FileConflictAction.IgnoreAll;
         }
 
-        public PackageExtractionContext PackageExtractionContext { get; set; } = new PackageExtractionContext();
+        public PackageExtractionContext PackageExtractionContext { get; set; } = new PackageExtractionContext(null);
```

Comment:
Promote the statement readability here: pass `NullLogger.Instance` or use named arg.